templates/articles/Estimating an Agile Projects Scope.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
```

Comment:
You should omit the document type declaration, `html` tag, `meta` tag, `title` tag, `head` closing tag, and `body` tag (basically everything up until line 8) because this document will be embedded.